### PR TITLE
Convert csv to list tables

### PR DESCRIPTION
**Pull request by Urhengulas**
Fixes https://app.clickup.com/t/8693umm77.

### CODE REVIEW

File: ferrocene/doc/evaluation-plan/src/method.rst [hunk 1]
```diff
@@ -21,15 +21,19 @@ Through this analysis, the following potential error are identified:
 Errors, with the possibility of detection and mitigation, are added in
 the :doc:`safety-manual:index` with the following form:
 
-.. csv-table:: Safety Manual  potential error template
+.. list-table:: Safety Manual  potential error template
    :name: Safety Manual potential error template
```
Comment (Urhengulas) [inline ferrocene/doc/evaluation-plan/src/method.rst:25]: Does the name need to be duplicate?

Comment (pietroalbini) [reply ferrocene/doc/evaluation-plan/src/method.rst:25]: I mean, I'd personally remove table names completely. It doesn't need to be duplicated though.

Comment (Urhengulas) [reply ferrocene/doc/evaluation-plan/src/method.rst]: Removed them

Comment (pietroalbini) [reply ferrocene/doc/evaluation-plan/src/method.rst]: There are still some table names in the diff (they're not duplicated though).

Comment (Urhengulas) [reply ferrocene/doc/evaluation-plan/src/method.rst]: Removed them as well

**Review by pietroalbini — APPROVED** [review]
bors merge

**Pull request merged**